data/remotes.d/fwupd.conf

```diff
@@ -0,0 +1,6 @@
+[fwupd Remote]
+
+# this remote provides metadata shipped with the fwupd package
+Enabled=true
+Type=local
+File=@datadir@/fwupd/remotes.d/fwupd/metadata.xml
```

Comment:
Any reason not to compress this metadata?  (Although keep in mind that if you choose to compress it that it needs to be added here too: https://github.com/hughsie/fwupd/blob/master/contrib/debian/rules#L41)